compiler/natives/src/crypto/subtle/subtle.go

```diff
@@ -36,3 +38,37 @@ func getSubtle() *js.Object {
 func init() {
 	SubtleCrypto = getSubtle()
 }
+
+// SubtleCall calls the object's method with the given arguments assuming that it returns a promise, wait for it to be settled, and return the result/error.
+func SubtleCall(method string, args ...interface{}) (*js.Object, error) {
+	if SubtleCrypto == nil {
+		return nil, ErrCryptoNotFound
+	}
+
+	// First check that the method exists (if it doesn't, Call() would panic, so it's better to check first)
```

Comment:
Not saying this is a bad approach, but an alternative would be to catch the panic with a `recover` in a deferred function. Doing so might also be a bit more robust, as it would catch any other runtime panics in an odd JS implementation or something.